src/execution_plan/ops/op_create.c

```diff
@@ -98,15 +140,26 @@ void _CreateNodes(OpCreate *op, Record r) {
 		PendingProperties *converted_properties = NULL;
 		if(map) converted_properties = _ConvertPropertyMap(r, map);
 
+		/* If we're only inserting unique entities, verify that this entity is new. */
+		if(op->unique_entities &&
+		   !_EntityIsDistinct(op->unique_entities, GETYPE_NODE, n->labelID, converted_properties)) {
+			_PendingPropertiesFree(converted_properties);
+			continue;
```

Comment:
I think this is problematic, consider an empty graph
`UNWIND [1, 2] AS x MERGE ({v:x})-[:e]->(n {v:1})`
The first iteration would create `n`
But what would happen in the second iteration `n` is no longer distinct.

I'm expecting 4 nodes and 2 edges in total to be created.